pkg/cli/mirror/fbc_operators.go

```diff
@@ -462,15 +473,22 @@ func addCatalogToMapping(catalogMapping image.TypedImageMapping, srcOperator v1a
 
 	ctlgSrcTI := image.TypedImage{
 		TypedImageReference: ctlgSrcTIR,
-		OriginalRef:         srcOperator.OriginalRef,
 		Category:            v1alpha2.TypeOperatorCatalog,
 	}
 
 	ctlgDstTI := image.TypedImage{
 		TypedImageReference: ctlgDstTIR,
-		OriginalRef:         srcOperator.OriginalRef,
 		Category:            v1alpha2.TypeOperatorCatalog,
 	}
+
+	if image.IsFBCOCI(srcCtlgRef) {
+		ctlgSrcTI.ImageFormat = image.OCIFormat
+		ctlgDstTI.ImageFormat = image.OCIFormat
+	} else {
+		ctlgSrcTI.ImageFormat = image.OtherFormat
```

Comment:
Is there a specific reason to set this value to other format? Could it left unset or is the OtherFormat being used somewhere else.